tests/system/testdata/endtoend/http.txtar

```diff
@@ -0,0 +1,128 @@
+ak manifest apply project.yaml
+return code == 0
+
+ak project build myproject --from main.star
+return code == 0
+output equals 'build_id: bld_00000000000000000000000006'
+
+ak deployment create --build-id=bld_00000000000000000000000006 --env=myproject/default --activate
+return code == 0
```

Comment:
Nit: you can use "ak project deploy", because it's not critical for this PR because I'll replace "ak manifest apply" and this anyway with "ak deploy".